### PR TITLE
fix: scroll on nft marketplace for mobile

### LINKED ISSUE #1059: Mobile: Buy NFT screen
Issues;
Buy NFT:
- Can’t scroll down 

Marketplace collection screen 
- Can’t scroll down. Layout issues

I believe this is a regression, at least scroll down was working.

### PR DESCRIPTION
**Pull request by sachin-into**
fixes #1059

### CODE REVIEW

**Review by omniwired — DISMISSED** [review]

File: packages/components/nfts/NFTs.tsx [hunk 1]
```diff
@@ -66,6 +66,7 @@ export const NFTs: React.FC<{
           )}
           minElemWidth={250}
           gap={layout.spacing_x2}
+          noFixedHeight={Platform.OS !== "web"}
```
Comment (n0izn0iz) [inline packages/components/nfts/NFTs.tsx:69]: this will load up to 10000 nfts, we can't do this here!

**Review by n0izn0iz — CHANGES_REQUESTED** [review]

File: packages/components/nfts/NFTs.tsx (hunk 1 above)
Comment (sachin-into) [reply packages/components/nfts/NFTs.tsx:69]: ok

File: packages/components/nfts/NFTs.tsx [hunk 2]
```diff
@@ -53,21 +52,38 @@ export const NFTs: React.FC<{
...
+                data={info.item}
+                style={{ width: elemWidth }}
+              />
+            )}
+            minElemWidth={250}
+            gap={layout.spacing_x2}
+          />
+        ) : (
+          <FlatList
+            data={nfts}
```
Comment (sachin-into) [inline packages/components/nfts/NFTs.tsx:72]: Since header and other options are on other components. Just created FlatList for mobile

Comment (n0izn0iz) [reply packages/components/nfts/NFTs.tsx:72]: I think this is the same behavior, gridlist is already a flatlist, with noFixedHeight it's like the flatlist you added, did you check network request to see if it does not load everything?

Comment (sachin-into) [reply packages/components/nfts/NFTs.tsx:72]: Yeah, it's same.
I did check on item length and it was fetching next 100 item at a time.

Comment (omniwired) [reply packages/components/nfts/NFTs.tsx:72]: Thanks for confirming

**Pull request merged**